src/sequelize/models/models.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Nice work @habinezadalvan, however, I was wondering if you can try to solve this ESlint issues instead of ignoring them. For example, you should create separate files for each model rather than create a single file for all models.

Reply:
Thanks @harerakalex, I worked on your feedback.